service/keyv3/key.go

```diff
@@ -128,7 +128,6 @@ func MemoryQuantity(n v1alpha1.KVMConfigSpecKVMNode) (resource.Quantity, error)
 		return resource.Quantity{}, microerror.Maskf(err, "creating Memory quantity from addtional memory")
 	}
 	q.Add(additionalMemory)
-	q.String()
```

Comment:
This is actually an issue for `go vet`. It does also not make that much sense to call a getter here without using its return value. Fixing the lazy hack is the way to go. I realized why we did this though, nevertheless it is wrong. See also https://github.com/kubernetes/apimachinery/issues/34.

Reply:
:+1: